src/client/components/features/Engagement/Rechercher/FormulaireRecherche/FormulaireRechercheMissionEngagement.test.tsx

```diff
@@ -83,20 +89,19 @@ describe('FormulaireRechercheMissionEngagement', () => {
 			);
 
 			const user = userEvent.setup();
-			const inputCommune = screen.getByTestId('InputCommune');
-			await user.type(inputCommune, 'Pari');
-			const résultatsCommune = await screen.findByTestId('RésultatsCommune');
-			const resultListCommune = within(résultatsCommune).getAllByRole('option');
-			fireEvent.click(resultListCommune[0]);
+			const comboboxCommune = screen.getByRole('combobox', { name: 'Localisation' });
+			await user.type(comboboxCommune, 'Pari');
+			const resultListCommune = screen.getAllByRole('option');
+			await user.click(resultListCommune[0]);
```

Comment:
Tu as une assertion sur un élément en particulier ("Paris (75006)"), je préfèrerait du coup cliquer sur cet élément-là plutôt que simplement "le premier" :
```suggestion
			const optionParis = screen.getByRole('option', { name: /Paris (75006)/i });
			await user.click(optionParis);
```
voire (tester si ça fonctionne) un [`selectOptions`](https://testing-library.com/docs/user-event/utility#-selectoptions-deselectoptions)
```suggestion
			const listeSuggestions = screen.getByRole('listbox');
			await user.selectOptions(listeSuggestions, 'Paris (75006)');
```
Je pense jamais à l'utiliser le `selectOptions` mais il est quand même pas mal pour le coup, il fait des trucs intelligents

Reply:
sympa selectOptions je connaissais pas, trop cool